### ISSUE TITLE
Help with installation of rimusz/nfs-client-provisioner with helm v3

### ISSUE THREAD

**Issue by razbensimon** [rimusz/charts#109]
Is this a request for help?:
Yes

**Version of Helm and Kubernetes**: helm v3.5, kubectl: v1.21.2

**Which chart**:  rimusz/nfs-client-provisioner

**What happened**:
Error on installation
```
Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2"
```

**What you expected to happen**:
helm v3 install the chart

**How to reproduce it** (as minimally and precisely as possible):
`helm install nfs-us-central1-c rimusz/nfs-client-provisioner --namespace nfs-storage  --set nfs.server=${FSADDR} --create-namespace`

**Anything else we need to know**:
No

**Comment by rimusz** [comment]
Thanks for reporting the issue, it is not the helm 3 issue but deprecated `Deployment` API version used there.
I will try release new chart version soon.

**Comment by rimusz** [comment]
fixed in #110
